### DISCUSSION TITLE
Can I control the Complete job?

### DISCUSSION THREAD

**Discussion by eck-song** [community/community#126421] [Actions]
### Select Topic Area

Question

### Body

I want the process executed in the job to keep the execution in the next job.

Complete job is killing the process even if running with nohup. I want to know how to exclude complete jobs from specific jobs or keep the process alive.

- runner is macOS

<img width="470" alt="image" src="https://github.com/community/community/assets/87297532/b4cc1d62-1c44-48cc-a646-8be82ae38abe">

**Comment by CopperEagle** [comment]
Hi @SongQA 

As far as I can tell this probably means you want to extend the existing job to do further tasks. AFAIK, jobs really are meant to run independent from each other.

**Reply by eck-song** [reply]
Thank you answer. but i need this function.. 😭

**Reply by CopperEagle** [reply]
Can't you extend the job you already have at hand? Just add more tasks there.

**Reply by grandmas-favorite** [reply]
I would have to agree with @CopperEagle here - adding more tasks to the job that is already running would probably be the most efficient way to resolve this use case!

**Reply by eck-song** [reply]
Thank you for your answer.
We decided to change the implementation strategy to run independently.

I will close this discussion.
